selectors.go

```diff
@@ -250,3 +251,14 @@ func GetChainDetailsByChainIDAndFamily(chainID string, family string) (ChainDeta
 		return ChainDetails{}, fmt.Errorf("family %s is not yet support", family)
 	}
 }
+
+// ExtractNetworkName extracts the network name from a given selector string.
```

Comment:
I don't see any chain selector has network name embedded. Maybe you mean chainID ?